### ISSUE TITLE
I don't understand the mistake

### ISSUE THREAD

**Issue by izukiyoto** [xixi52/discord-canvas#58]
**rank.js**

```
const Discord = require("discord.js");
const Canvas = require("discord-canvas");

module.exports = {
  name: "rank",
  description: "Permet de pouvoir voir l'Expérience une personne",
  permission: "Aucune",
  dm: false,
  category: "Expérience",
  options: [
    {
      type: "user",
      name: "membre",
      description: "La personne à bannir",
      required: true,
    },
  ],

  async run(bot, message, args, db) {
    let user;
    if (args.getUser("membre")) {
      user = args.getUser("membre");
      if (!user || !message.guild.members.cache.get(user?.id))
        return message.replay("Pas de membre");
    } else user = message.user;

    db.query(
      `SELECT * FROM xp WHERE guild = '${message.guildId}' AND user = '${user.id}'`,
      async (err, req) => {
        db.query(
          `SELECT * FROM xp WHERE guild = '${message.guildId}'`,
          async (err, all) => {
            if (req.length < 1)
              return message.reply("Ce membre n'a pas encore d'xp");

            await message.deferReply();

            const calculXp = (xp, level) => {
              let xptotal = 0;
              for (i = 0; i < level + 1; i++) xptotal += i * 1000;
              xptotal += xp;
              return xptotal;
            };

            let leaderboard = await all.sort(async (a, b) =>
              calculXp(
                parseInt(b.xp),
                parseInt(b.level) - calculXp(parseInt(a.xp), parseInt(a.level))
              )
            );

            let level = parseInt(req[0].level);
            let xp = parseInt(req[0].xp);
            let rank = leaderboard.findIndex((r) => r.user === user.id) + 1;
            let need = (level + 1) * 1000;

            let Card = await new Canvas.RankCard()

              .setBackground(
                "https://images.hdqwalls.com/download/anime-sky-4k-0f-1920x1080.jpg"
              )
              .setAddon(rank)
              .setRankName(need)
              .setAvatar(message.user.displayAvatarURL({ extension: "jpg" }))
              .setLevel(level)
              .setReputation(xp)
              .setUsername(user.tag)
              .toAttachment();

            await message.followUp({
              files: [
                new Discord.AttachmentBuilder(Card.toBuffer(), {
                  name: "rank.png",
                }),
              ],
            });
          }
        );
      }
    );
  },
};
```

**error**

```
\Tsuki\node_modules\discord-canvas\utils\functions.js:10
        const formattedVariable = variable.toLowerCase()
                                           ^

TypeError: variable.toLowerCase is not a function
    at formatVariable (C:\Users\Izuki\OneDrive\Documents\Tsuki\node_modules\discord-canvas\utils\functions.js:10:44)
    at RankCard.setAddon (C:\Users\Izuki\OneDrive\Documents\Tsuki\node_modules\discord-canvas\src\rank\Rank.js:107:31)
    at Query.<anonymous> (C:\Users\Izuki\OneDrive\Documents\Tsuki\Commandes\rank.js:62:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

**Comment by izukiyoto** [comment]
The problem comes from` .setAddon(rank)` and `.setRankName(need)`

**Comment by izukiyoto** [comment]
bye

**Comment by emidblol** [comment]
I have MAYBE fixed this in #59
